services/user/user-service/src/main/java/com/milesight/beaveriot/user/service/RoleService.java

```diff
@@ -390,11 +391,10 @@ public Page<RoleDashboardResponse> getDashboardsByRoleId(Long roleId, RoleDashbo
         });
     }
 
-    public List<DashboardUndistributedResponse> getUndistributedDashboards(Long roleId, DashboardUndistributedRequest dashboardUndistributedRequest) {
-        //TODO sort
-        List<DashboardDTO> dashboardDTOList = dashboardFacade.getDashboardsLike(dashboardUndistributedRequest.getKeyword());
+    public Page<DashboardUndistributedResponse> getUndistributedDashboards(Long roleId, DashboardUndistributedRequest dashboardUndistributedRequest) {
+        List<DashboardDTO> dashboardDTOList = dashboardFacade.getDashboardsLike(dashboardUndistributedRequest.getKeyword(), JsonUtils.toJSON(dashboardUndistributedRequest.getSort().toSort()));
```

Comment:
Why does Sort need to be converted to a string?

Reply:
fixed